Dockerfile

```diff
@@ -2,13 +2,16 @@ ARG BASE_IMAGE=node:16-buster
 FROM $BASE_IMAGE
 RUN apt update && apt-get install sqlite3
 RUN mkdir -p /harmony
-COPY ./package.json package-lock.json lerna.json /harmony/
+COPY package.json package-lock.json lerna.json /harmony/
+RUN chown node -R /harmony
+USER node
 WORKDIR /harmony
-RUN npm install
+RUN npm ci
+RUN npm install sqlite3 --save
 COPY . /harmony/
-RUN rm -f config/services.yml
 # build the sqlite dabase
+USER root
```

Comment:
Even though I copied in the harmony directory as the node user and ran the steps to install and copy in directories as the node user it would fail with permission issues, so I just work around it by switching back to the root user and then running chown afterwards on just the directories needed to be writeable.